### DISCUSSION TITLE
MediaElement support for full screen on Android

### DISCUSSION THREAD

**Discussion by JoacimWall** [CommunityToolkit/Maui#883] [New Feature Discussions]
MediaElement support for full screen on Android the same as in iOS. Maybe I missed something but can't find a way to go from list of movies to full screen as iOS supports it.

This guide may help 
 

https://github.com/Norulab/android-exoplayer-fullscreen
 
//Joacim

**Comment by EdwardMcFlurry** [comment]
When can we expect at least a release on this Nuget Package, currently I’m using the one from https://github.com/davidbritch/dotnet-maui-videoplayer but I’m stuck with Android and can’t figure out what wrong with it?

**Comment by CliffAgius** [comment]
Closed and answered
